### PR TITLE
Update meta.yaml

### PR DESCRIPTION
**Pull request by satra**

### CODE REVIEW

**Comment by satra** [comment]
@jakirkham - any help here would be much appreciated. i cannot figure out why the travis and circle builds are failing (looks like for different reasons).

**Comment by satra** [comment]
also not sure why the new conda-smithy update removes py3x from travis and appveyor.

**Comment by jakirkham** [comment]
Am AFK. Would recommend asking others.

**Comment by isuruf** [comment]
Travis and CircleCI are both failing for the same reason. `setuptools` is trying to download `configparser`, but dependencies should be in meta.yaml

Can you please close this PR and send a new PR from your fork? Branches of the main repo are used rarely and only for maintaining different versions.

**Comment by satra** [comment]
@isuruf - meta.yml has this line:

- configparser  # [py27 or py34]

**Comment by isuruf** [comment]
Yes, but the error happens on py36

**Comment by satra** [comment]
how should i tell it not to use configparser for py36 or py35? (doesn't the [py27 pr py34] restrict it to those two versions? )

the travis error is with py27

also any thoughts on why conda-smithy removed py3x from travis?

**Comment by satra** [comment]
on the travis log, it looks like it's downloading things for py27 and then suddenly switches to py36 version of setuptools.

**Comment by isuruf** [comment]
> how should i tell it not to use configparser for py36 or py35? (doesn't the [py27 pr py34] restrict it to those two versions? )

This is not conda's fault. The package has configparser as a requirement. See here, https://github.com/nipy/nipype/blob/master/nipype/info.py#L141 . Can you ask upstream to have `configparser` as a requirement only for python <= 3.4

> the travis error is with py27
Travis log is messed up, but the problem is in 3.6 there as well.

**Comment by satra** [comment]
@isuruf - i'll change that upstream. will need to make a new release though. since it installs fine with normal pip and python setup, we didn't bother restricting it.

**Comment by satra** [comment]
@isuruf - to test out your suggestion i implemented a branch that adjusts setup.py requirements. and it results in the same error. as seen here: https://github.com/nipy/nipype/pull/2015/files

i believe the issue may be with the requirements.txt file in the repo, which cannot be changed dynamically. if you do `python setup.py install` or `pip install .` it works fine with the current released version of code on all supported platforms (py27, py3[4-6]. so i don't understand why conda build doesn't.

this surely seems like a conda issue at this point. would be happy to test other suggestions.

**Comment by satra** [comment]
also as a note configparser exists in the previous release (i.e. the current feedstock), and it build fine for multiple platforms.

**Comment by isuruf** [comment]
Then you can just delete the requirements.txt before installing.

Also can you send the PR from a fork?

**Comment by satra** [comment]
> Then you can just delete the requirements.txt before installing.

how do i do that with meta.yml? do i use a build.sh?

> Also can you send the PR from a fork?

i'll do that one i figure that this works.

why did the previous released feedstock work? was this change implemented recently?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,47 +7,55 @@ package:
...
   fn: {{ version }}.tar.gz
   url: https://github.com/nipy/nipype/archive/{{ version }}.tar.gz
-  md5: 69a0de06049c694ffaafeed80b2037a1
+  md5: b8d51c74661e92812fbb01f0a6397c49
 
 build:
-  number: 1
-  script: python setup.py install
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (isuruf) [inline recipe/meta.yaml]: You can change this to,
```
script:
 - rm requirements.txt
 - python setup.py install --single-version-externally-managed --record=record.txt
```

**Comment by isuruf** [comment]
> i'll do that one i figure that this works.

Note that if a build succeeds, it is immediately uploaded if you are sending PRs from the main repo. So, if this succeeds, there's no point sending a new PR. Also notice that travis-ci and appveyor is run twice which is a waste of resources.

**Comment by satra** [comment]
@isuruf - thanks for the clarification on the build process.

**Comment by isuruf** [comment]
This line needs to be fixed as well, https://github.com/nipy/nipype/blob/master/setup.py#L137

**Comment by satra** [comment]
@isuruf- yes 

i was able to run the conda build with the conda-forge docker container and build things on a local build. once we cut a new release, i will push the changes.

**Pull request closed without merging**